extension/src/project-explorer/new-project.ts

```diff
@@ -1,11 +1,17 @@
 import path from 'path';
 import * as vscode from 'vscode';
+import { logInformationMessage } from '../base/logging-util';
 import { IvyEngineManager } from '../engine/engine-manager';
 import { TreeSelection, treeSelectionToUri } from './tree-selection';
 import { validateArtifactName, validateDotSeparatedName } from './util';
 
 export const addNewProject = async (selection: TreeSelection) => {
-  const selectedUri = await treeSelectionToUri(selection);
+  const selectedUri = (await treeSelectionToUri(selection)) ?? vscode.workspace.workspaceFolders?.at(0)?.uri;
```

Comment:
I think the copy clipboard hack can be fixed here. just verify that `await treeSelectionToUri(selection)` is a directory.

can be done in a followup or I can also take over. your choice :)